fcrepo6/rootfs/opt/tomcat/bin/setenv.sh

```diff
@@ -32,3 +32,14 @@ fi
 if [[ "${FCREPO_DISABLE_SYN}" == "true" ]]; then
   export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.properties.management=relaxed"
 fi
+
+if [[ "${FCREPO_BINARYSTORAGE_TYPE}" == "file" ]]; then
+  export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.storage=ocfl-fs"
+fi
+if [[ "${FCREPO_BINARYSTORAGE_TYPE}" == "s3" ]]; then
+  # Export AWS credentials
+  export AWS_ACCESS_KEY_ID=${FCREPO_S3_USER}
+  export AWS_SECRET_ACCESS_KEY=${FCREPO_S3_PASSWORD}
+  # Enable S3 mode and set default options
+  export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.storage=ocfl-s3 -Dfcrepo.aws.region=${FCREPO_AWS_REGION} -Dfcrepo.aws.region=${FCREPO_S3_BUCKET} -Dfcrepo.aws.region=${FCREPO_S3_PREFIX}"
```

Comment:
Should `-Dfcrepo.aws.region` be specified with three different values (`FCREPO_AWS_REGION`, `FCREPO_S3_BUCKET` and `FCREPO_S3_PREFIX`) or should each of the environment variables be passed to a different Java property?